actionsystemtest/lib/action_system_test.rb

```diff
@@ -0,0 +1,67 @@
+# System tests are similar to Integration tests in that they incorporate multiple
+# controllers and actions, but can be used to simulate a real user experience.
+# System tests are also known as Acceptance tests.
+#
+# To create a System Test in your application, extend your test class from
+# <tt>ActionSystemTestCase</tt>. System tests use Capybara as a base and
+# allow you to configure the driver. The default driver is
+# <tt>RailsSeleniumDriver</tt> which provides a Capybara and the Selenium
+# Driver with no configuration. It's intended to work out of the box.
+#
+# A system test looks like the following:
+#
+#   require 'system_test_helper'
+#
+#   class Users::CreateTest < ActionSystemTestCase
+#     def adding_a_new_user
+#       visit users_path
+#       click_on 'New User'
+#
+#       fill_in 'Name', with: 'Arya'
+#       click_on 'Create User'
+#
+#       assert_text 'Arya'
+#     end
+#   end
+#
+# When generating an application or scaffold a +system_test_helper.rb+ will also
+# be generated containing the base class for system testing. This is where you can
+# change the driver, add Capybara settings, and other configuration for your system
+# tests.
+#
+#   class ActionSystemTestCase < ActionSystemTest::Base
+#     ActionSystemTest.driver = :rack_test
```

Comment:
I don't think we have a compelling argument to ship with a Rack driver or highlight its availability. Since the only reason you'd suffer the slowdown of running through a browser for testing is to get the JavaScript tested too, I don't think it makes sense to promote a rack way that doesn't allow this.

I'd vote to nix this option. If there's a backdoor somewhere to do it by setting things by hand, fine. But it shouldn't be promoted and we should make no special accommodations for it.